### DISCUSSION TITLE
unexperienced user - any advice where to start learning or step by step explanation?

### DISCUSSION THREAD

**Discussion by LK2408** [mxstbr/karabiner#26] [Q&A] [answered]
I saw this on youtube and would love to set this up. I am willing to spent a lot of hours on learning things to set this up myself or would love to get a step by step tutorial. I know nothing about coding, I don't know what a github symlink is, how I clone a repository etc.

**Comment by mxstbr** [comment, marked as answer]
Hi @LK2408, thanks for reaching out. Truthfully, you're going to have a difficult time using this setup as it requires coding to use, I really didn't make it for somebody that doesn't know how to code.

I would recommend looking into tools like KeyboardMaestro, hyperkey.app, or BetterTouchTool as non-coding-required alternatives. They don't quite do the exact same thing, but can get you close.
